larray/viewer/api.py

```diff
@@ -107,16 +108,16 @@ def edit(obj=None, title='', minvalue=None, maxvalue=None, readonly=False, depth
         local_vars = sys._getframe(depth + 1).f_locals
         obj = OrderedDict([(k, local_vars[k]) for k in sorted(local_vars.keys())])
 
-    if isinstance(obj, str):
-        if os.path.exists(obj):
-            obj = la.Session(obj)
-        else:
-            raise ValueError("file {} not found".format(obj))
+    if isinstance(obj, int) and obj == -2:
+        obj = None
```

Comment:
Reusing None for something else seems like a bad idea to me. Having one variable mean two different things for the same value depending on where in the code you are seems like a bad practice to me.